src/browser/provider/built-in/path.js

```diff
@@ -44,6 +44,8 @@ export default class PathBrowserProvider extends BrowserProviderBase {
 
         if (OS.win)
             await super.calculateResizeCorrections(browserId);
+        else if (OS.mac)
```

Comment:
Shouldn't it be in base provider? (like resize corrections). I wonder how I can use this functionality in my browser provider (e.g. for redistributable browsers)